entgql/pagination.go

```diff
@@ -242,34 +302,66 @@ func multiPredicate[T any](cursor *Cursor[T], opts *MultiCursorsOptions) (func(*
 					ands = append(ands, sql.EQ(s.C(column), values[j]))
 				}
 			}
-			// TODO: Whenever nulls last is made configurable, we'll want to change this behavior. Sorting works differently
-			// when nulls last isn't specified. This code works for nulls last because that's the behavior we want, but before
-			// we can merge it back we need to spend some time making it configurable. Without nulls first nulls will appear
-			// first when in desc order, and last when ordering by asc order. So when we can expect to have values appear first
-			// and nulls appear second, we need to ad an `or x is null` when the cursor has a value. When using `nulls last`
-			// this is always the case, nulls are always last. When not using nulls last then we have to account for the
-			// possibility of nulls appearing first when sorting in descending order, which would mean we have to add an
-			// `or x is not null` so that we don't stop paginating early. This conflicts with the behavior we want with
-			// nulls last.
 			if opts.Directions[i] == OrderDirectionAsc {
-				if values[i] != nil {
-					// we have a value and we're sorting in ascending order, that means that we need to add `or x is null` so that
-					// pagination doesn't stop early, because > null doesn't work. If we add support for `nulls first` then this
-					// will need to change to accomodate that.
+				switch {
```

Comment:
Naked switch, this should probably be `switch values[i]`, or just if statements.

This is more complicated than I was expecting, why are there so many conditions?